### PR TITLE
[13_2] 2023 FastSim

### PR DESCRIPTION
**Pull request by srimanob**
#### PR description:
Backport of https://github.com/cms-sw/cmssw/pull/42327 (overwrite https://github.com/cms-sw/cmssw/pull/42295)

### CODE REVIEW

**Comment by srimanob** [comment]
test parameters:
- workflows = 14034.0,14234.0
- relvals_opt = --what cleanedupgrade,standard,highstats,pileup,generator,extendedgen,production,identity,ged,machine,premix,nano,gpu,2017,2026

**Comment by srimanob** [comment]
+Upgrade

**Comment by perrotta** [comment]
+1

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -4042,15 +4043,13 @@ def gen2023HiMix(fragment,howMuch):
     name=baseDataSetReleaseBetter[key]
     if '2017' in ds:
         PUDataSets[ds]={'-n':10,'--pileup':'AVE_35_BX_25ns','--pileup_input':'das:/RelValMinBias_13/%s/GEN-SIM'%(name,)}
-    elif '2018' in ds:
+    elif '2018' in ds or 'postLS2' in ds:
         PUDataSets[ds]={'-n':10,'--pileup':'AVE_50_BX_25ns','--pileup_input':'das:/RelValMinBias_13/%s/GEN-SIM'%(name,)}
-    elif '2021' in ds:
+    elif '2021' in ds or '2023' in ds:
```
Comment (slava77) [inline Configuration/PyReleaseValidation/python/relval_steps.py:4048]: @srimanob 
was there a good reason to not update 2024 workflows? (they are still stuck at PU35)

**Pull request merged**